deployment/network-operator/templates/sriovnetwork.openshift.io_sriovoperatorconfig.yaml

```diff
@@ -24,8 +24,8 @@ spec:
   enableInjector: false
   enableOperatorWebhook: false
   configDaemonNodeSelector:
-    beta.kubernetes.io/os: linux
-    node-role.kubernetes.io/worker: ""
-    network.nvidia.com/operator.mofed.wait: "false"
+  {{- with .Values.sriovNetworkOperator.configDaemonNodeSelector }}
```

Comment:
I would like to leave these two selectors because they're mandatory to get everything work:
beta.kubernetes.io/os: linux
network.nvidia.com/operator.mofed.wait: "false"